src/topp/DecoyDatabase.cpp

```diff
@@ -16,6 +16,13 @@
 #include <OpenMS/MATH/MathFunctions.h>
 #include <OpenMS/DATASTRUCTURES/FASTAContainer.h>
 #include <regex>
+#include <OpenMS/ANALYSIS/ID/NeighborSeq.h>
+#include <OpenMS/CHEMISTRY/TheoreticalSpectrumGenerator.h>
+#include <OpenMS/KERNEL/MSSpectrum.h>
+#include <cmath>
+#include <set>
```

Comment:
no need for these?

Reply:
#include <OpenMS/CHEMISTRY/TheoreticalSpectrumGenerator.h>
#include <OpenMS/KERNEL/MSSpectrum.h>
I need for the spectrum generation 
#include <cmath> I need for the abs

Reply:
but not in DecoyDatabase.cpp